src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts

```diff
@@ -122,16 +124,51 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
    * This builds the form with data from the previous pages
    * EUI-3732 - Breathing space data not persisted on Previous button click with ExpUI Demo
    */
-  toPreviousPage() {
+  public toPreviousPage(): void {
+    this.validationErrors = [];
     let caseEventData: CaseEventData = this.buildCaseEventData();
     caseEventData.data = caseEventData.event_data;
     this.updateFormData(caseEventData);
     this.previous();
     CaseEditPageComponent.setFocusToTop();
   }
 
-  submit() {
-    if (!this.isSubmitting) {
+  // Adding validation message to show it as Error Summary
+  generateErrorMessage(fields: CaseField[]): void {
```

Comment:
_Nitpick:_
Sorry, @RiteshHMCTS, I should also have said `public` in my previous comment on this one.

Reply:
updated